README.md

```diff
@@ -175,6 +175,12 @@ Prowadzący Ijon Tichy/marszaukowska
 
 Temat: Zrób se pager, aparat i kalendarz - warsztat z raspberry pi, od podstaw. Coś zepsujemy, coś naprawimy - będzie fajnie ;)
 
+14.03.2021
+
+Prowadzący Mateusz Bełczowski / nikisaku
+
+Temat: Dodanie do Oprheusa przypisywania ról na podstawie kliknięcia w reakcję pod postem.
```

Comment:
`Orpheusa`?

Reply:
To miał być zupełnie osobny bot? Ja zrozumiałem, że mamy naszego wzbogacić.

Reply:
No naszego. Tylko do litrówki się przyczepiłem ;)